src/OrbitVulkanLayer/VulkanLayerController.h

```diff
@@ -460,6 +500,73 @@ class VulkanLayerController {
     }
   }
 
+  template <typename T>
+  void AddRequiredExtensionNameIfMissing(
+      const T* create_info,
+      const std::function<VkResult(uint32_t*, VkExtensionProperties*)>&
+          enumerate_extension_properties_function,
+      const char* extension_name, std::vector<std::string>* output) {
+    bool host_query_reset_extension_enabled = false;
+    for (uint32_t i = 0; i < create_info->enabledExtensionCount; ++i) {
+      const char* enabled_extension = create_info->ppEnabledExtensionNames[i];
+      if (strcmp(enabled_extension, extension_name) == 0) {
+        host_query_reset_extension_enabled = true;
+        break;
+      }
+    }
+    if (!host_query_reset_extension_enabled) {
```

Comment:
```suggestion
    }
    
    if (!host_query_reset_extension_enabled) {
```

This method would benefit from some additinal vertical space in general.